sneck/Deck.py

```diff
@@ -528,6 +733,37 @@ def update(self, stack: dict) -> None:
         
         print(f'[UPDATE]: Updated stack {self.__id}.')
 
+    def add_card(self, card: DeckCard) -> None:
```

Comment:
I like the approach with the serialize, I do think it would be more convenient to just call `update()` on `self` in this case rather than adding the missing pieces manually for a couple of reasons

1. Avoids code duplication (since the `__init__` of `DeckCard` already does all of this)
2. Avoids desynchronization issues in case something goes wrong during the POST or input sanitisation is performed server side thus resulting in a change of the posted data
3. Increments chances of timely catching other updates in case multiple edits are being made